leshan-core/src/main/java/org/eclipse/leshan/json/JsonArrayElement.java

```diff
@@ -0,0 +1,167 @@
+/*******************************************************************************
+ * Copyright (c) 2013-2015 Sierra Wireless and others.
+ * 
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v1.0 which accompany this distribution.
+ * 
+ * The Eclipse Public License is available at
+ *    http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ *    http://www.eclipse.org/org/documents/edl-v10.html.
+ * 
+ * Contributors:
+ *     Sierra Wireless - initial API and implementation
+ *     Gemalto M2M GmbH
+ *******************************************************************************/
+
+package org.eclipse.leshan.json;
+
+import com.google.gson.annotations.SerializedName;
+
+public class JsonArrayElement{
+
+	@SerializedName("n")
```

Comment:
yet again formatting,

if you do what written in the README it should work: just "mvn eclipse:eclipse" and import the project.